qemu/tests/virtio_port_login.py

```diff
@@ -22,7 +24,7 @@ def __init__(self, test, env, params):
 
     @error_context.context_aware
     def pre_step(self):
-        error_context.context("Config guest and reboot it", logging.info)
+        error_context.context("Config guest and reboot it", LOG_JOB.info)
```

Comment:
It can be replaced with `self.test.log.info` as `utils_virtio_port.VirtioPortTest` support `self.test` in `__init__`.

Reply:
Same for others in this class